File: network/pyjava_servers/TCPClient.py
#!/usr/bin/python
#TCPClient.py
from socket import *
serverName = "127.0.0.1"
serverPort = 7777
clientSocket = socket(AF_INET, SOCK_STREAM)

clientSocket.connect((serverName,serverPort))
sentence = input("Input lowercase sentence: ")
clientSocket.send((sentence+'\n').encode())
modifiedSentence = clientSocket.recv(1024)
print("From  Server: ", modifiedSentence.decode('utf-8'))
clientSocket.close()

File: network/pyjava_servers/TCPClient.java
import java.io.*;
import java.net.*;

class TCPClient{
    public static void main(String[] argv)throws Exception{
	String sentence;
	String modifiedSentence;
	BufferedReader inFromUser=new BufferedReader(new InputStreamReader(System.in));
	Socket clientSocket=new Socket("localhost",7777);
	DataOutputStream outToServer=new DataOutputStream(clientSocket.getOutputStream());
	BufferedReader inFromServer=new BufferedReader(new InputStreamReader(clientSocket.getInputStream()));
	sentence = inFromUser.readLine();
	outToServer.writeUTF(sentence+"\n");
	modifiedSentence=inFromServer.readLine();
	System.out.println("From Server----->"+modifiedSentence);
	clientSocket.close();
    }
}
